client/src/main/java/com/flipkart/flux/client/config/FluxClientConfiguration.java

```diff
@@ -31,16 +31,20 @@
 public class FluxClientConfiguration {
 
     private String fluxRuntimeUrl = "http://localhost:9998";
+    private String clientElbId = "client_elb_id_1";
```

Comment:
Is this the default clientElbId ? If it is use a better name like "defaultElb" or somethin, which should be a global constant as well.

Reply:
Added "defaultElbId" as a default name.